### PR TITLE
[xml] Update german.xml

### PR DESCRIPTION
**Pull request by schnurlos**
Following https://github.com/notepad-plus-plus/notepad-plus-plus/commit/2c1212cba5372b9777d583810c981a8c725fd18f & https://github.com/notepad-plus-plus/notepad-plus-plus/commit/287c85f8f0ba792c538f633205e10976d815da02

### CODE REVIEW

**Review by donho — CHANGES_REQUESTED** [review]
Please add a new entry also in Tabbar context menu, as latest commit in english.xml:
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/f18d99e24cb3609f912e7c7bf0f1d3de2b48268f

**Comment by schnurlos** [comment]
@donho done

File: PowerEditor/installer/nativeLang/german.xml [hunk 1]
```diff
@@ -431,6 +431,7 @@ Translation note:
 				<Item CMDID="41009" name="Dateien links von aktueller Datei schlie&amp;ßen"/>
 				<Item CMDID="41018" name="Dateien rech&amp;ts von aktueller Datei schließen"/>
 				<Item CMDID="41024" name="Alle unveränderten Dateien schließen"/>
+				<Item CMDID="44048" name="Tab Fixieren" alternativeName="Tab Lösen"/>
```
Comment (CennoxX) [inline PowerEditor/installer/nativeLang/german.xml:436]: The verbs should be written in lower case: "Tab fixieren" and "Tab lösen"

File: PowerEditor/installer/nativeLang/german.xml [hunk 2]
```diff
@@ -963,11 +964,11 @@ Translation note:
 					<Item id="6106" name="Dateireiter (Tabs)"/>
 					<Item id="6107" name="Verkleinern"/>
 					<Item id="6108" name="Tabs nicht mit der Maus verschiebbar"/>
-					<Item id="6109" name="Inaktive Tabs dunkler darstellen"/>
+					<Item id="6109" name="Farbe für Inaktive Tabs ändern"/>
```
Comment (CennoxX) [inline PowerEditor/installer/nativeLang/german.xml:969]: "Inaktive Tabs" should be lower case: "inaktive Tabs" as it's not at the beginning of a sentence. Or a slightly different wording: "Farbe inaktiver Tabs ändern"

**Review by donho — APPROVED** [review]

**Pull request closed without merging**